assets/javascripts/modules/analytics/setup.js

```diff
@@ -1,34 +1,37 @@
-/*global guardian:true */
```

Comment:
Nope, we tell the linter we're in a browser context and so it knows about those globals. The global definitions are only for non-host globals (i.e., ones we define)